examples/lakefs/enterprise/values-external-aws.yaml

```diff
@@ -0,0 +1,71 @@
+lakefsConfig: |
```

Comment:
please re-use existing example (OIDC most common) for completness 
many of the values here are relevant only for local developemtn like localhost. 
So I suggest: 
Duplicate the [oidc example](https://github.com/treeverse/charts/blob/master/examples/lakefs/enterprise/values-oidc.yaml) and add the nessceary stuff for AWS. 
call it `values-oidc-aws-external-principals.yaml`